src/App.js

```diff
@@ -1,74 +1,33 @@
 import React, { useState } from 'react';
-import { Day, DayView } from './components/Day';
-import { RadioButton } from './components/RadioButton';
-import AppCSS from './App.module.css';
-import { Button } from './components/Button';
-const ROW_LENGTH = 96;
-const ROW_DATA = [];
-
-for (let col = 0; col < ROW_LENGTH; col++) {
-  const cell = {
-    id: col,
-    mode: 'active',
-    selected: false
-  };
-  ROW_DATA.push(cell);
-}
+import styles from './App.module.css';
+import { SaveDay } from './components/SaveDay';
+import { EditDay } from './components/EditDay';
+import { Home } from './components/Home';
```

Comment:
For an easier organization, usually, we have `.pages/` to old the components that are pages. We leave `.components/` for components that are reusable across multiple pages.